conformance/src/main/scala/za/co/absa/enceladus/conformance/CmdConfig.scala

```diff
@@ -96,8 +96,8 @@ object CmdConfig {
     opt[String]("folder-prefix").optional().action((value, config) =>
       config.copy(folderPrefix = Some(value))).text("Adds a folder prefix before the infoDateColumn")
 
-    opt[Unit]("experimental-mapping-rule").optional().action((value, config) =>
-      config.copy(experimentalMappingRule = true)).text("Use experimental optimized mapping conformance rule")
+    opt[Boolean]("experimental-mapping-rule").optional().action((value, config) =>
+      config.copy(experimentalMappingRule = Some(value))).text("Use experimental optimized mapping conformance rule")
```

Comment:
I would use `Option(value)` since a variable is being passed in and that variable could be `null`